### ISSUE TITLE
Issue with Numpy 1.22

### ISSUE THREAD

**Issue by ericnorway** [dgoeries/lttbc#22]
I ran into an issue with numpy 1.22.2 and lttbc 0.2.2. Downgrading to lttbc 0.2.1 or upgrading to numpy 1.24 solved the issue.

Exception has occurred: ImportError
numpy.core.multiarray failed to import
  File "C:\temp\lttbc stuff\main.py", line 1, in <module>
    import lttbc
ImportError: numpy.core.multiarray failed to import

**Comment by dgoeries** [comment]
thanks, I will have a look

**Comment by vnadhan** [comment]
It looks like lttbc==0.2.2 is not in the pip registry.

ERROR: Could not find a version that satisfies the requirement lttbc==0.2.2 (from versions: 0.1.9, 0.2.0, 0.2.1)
ERROR: No matching distribution found for lttbc==0.2.2

**Comment by dgoeries** [comment]
Yes, there is some trouble. I am sorry for that

**Comment by eyjay-ok** [comment]
DO you have any updates here? When trying to install lttbc via poetry I run into a problem
`ModuleNotFoundError: No module named 'numpy'`
Poetry hints: `Note: This error originates from the build backend, and is likely not a problem with poetry but with lttbc (0.2.1) not supporting PEP 517 builds. You can verify this by running 'pip wheel --use-pep517 "lttbc (==0.2.1)"'.`

Is it possible that this error is also solved with the newer version.

I am using a Mac and lttbc don't have wheels for MacOS.

**Comment by dgoeries** [comment]
There will be a new version fixing this. I had a version up that unfortunately was always taken the latest numpy available when specified in the build requirements, providing incompatibilities.

Will try again and then upload a new version, sorry for inconvenience

**Comment by dgoeries** [comment]
Running through a lot of documents I could not fully find a satisfying solution. I went for pinning numpy 1.22 in the build process. Specifying a build requirement with _>=_ always did lead to the latest being used.
This resulted in incompatible versions. Googling and looking around, it looks like this package is not the only one suffering from this. Numpy has a large backward compatibility ...

Due to Python 3.11 support I went for 1.22 as requirement.
